src/tslint-cli.ts

```diff
@@ -81,6 +85,17 @@ if (argv.v != null) {
     process.exit(0);
 }
 
+if (argv.i != null) {
+    if (fs.existsSync(CONFIG_FILENAME)) {
+        console.error(`Cannot generate ${CONFIG_FILENAME}: file already exists`);
+        process.exit(1);
+    }
+
+    const tslintJSON = JSON.stringify(DEFAULT_CONFIG, undefined, "   ");
```

Comment:
> JS stdlib treats it as JSON

what does that mean?

The "JSON" substring feels a bit too direct of a conflict with the builtin `JSON` value.